package.json

```diff
@@ -29,6 +29,7 @@
     "async": "^1.5.2",
     "glob": "^6.0.4",
     "lodash": "^4.3.0",
+    "optimist": "^0.6.1",
```

Comment:
I had a quick looked at this nodejs module and the author says it was deprecated. He said that he is using minimist.
Wouldn't it be better using a non deprecated lib ?

Reference: https://github.com/substack/node-optimist